runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -1281,11 +1362,11 @@ func s:GotoAsmwinOrCreateIt()
     setlocal buftype=nofile
     setlocal modifiable
 
-    let asmbuf = bufnr('Termdebug-asm-listing')
-    if asmbuf > 0
-      exe 'buffer' . asmbuf
+    if s:asmbuf > 0
+      exe 'buffer' . s:asmbuf
```

Comment:
Use double dots for string concatenation and use a space after "buffer".

Reply:
Oops, I've thought those to be a typo and dropped them in #10393 - I guess it would be good to revert that and instead use it in every place that was actually changed, right?